docs/intro-to-eda.md

```diff
@@ -1,56 +1,53 @@
 ---
-title: Learn About Event Driven Architecture
+title: About Event Driven Architecture at VA
 ---
 
```

Comment:
In order to fix the problem of the table of contents not displaying on the right, we need to add `# About Event Driven Architecture at VA` at the top and make `Intro`, `Role within the VA enterprise environment`, `An opinionated conduit`, `Enterprise events versus data events`, `Technology overview`, and `Learn more` all H2.

Reply:
This is updated. Marking as resolved.